docassemble/RFApackage/data/questions/RFApackage.yml

```diff
@@ -1324,7 +1440,7 @@ fields:
     datatype: yesno
   - note: |
       <h2 class="h4">Race:</h2>
-  - "Select one.": race
```

Comment:
Its not just to remove the language, but actually allow user to select more than 1

Reply:
- [ ] https://github.com/VTskier/docassemble-RFApackage/issues/11